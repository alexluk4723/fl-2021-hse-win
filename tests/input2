b := ^j\^^*?+^abf^
c:=                 ^udjf^
a       :=c
d:=a|b|g
g:=^a^|^b^
return d
