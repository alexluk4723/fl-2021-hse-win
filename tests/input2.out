b := ^j\^^*?+^abf^
c:=                 ^udjf^
a       :=c
d:=a|b
return d
